TombEngine/Objects/TR3/Entity/tr3_wasp.cpp

```diff
@@ -0,0 +1,286 @@
+#include "framework.h"
+#include "Objects/TR3/Entity/tr3_wasp.h"
```

Comment:
Since these are new files, we can name them `Wasp.cpp` and `Wasp.h` from the start.

Reply:
It was Stranger which created it, will do that later.

Reply:
Cool. Let's do WaspMutant out of pedantry.

Reply:
Done :)